webhook/webhook.go

```diff
@@ -206,16 +221,38 @@ func (wh *Webhook) Run(stop <-chan struct{}) error {
 
 	select {
 	case <-stop:
-		if err := server.Close(); err != nil {
+		// Start failing readiness probes immediately.
+		logger.Debug("Starting to fail readiness probes...")
```

Comment:
```suggestion
		logger.Info("Starting to fail readiness probes...")
```
This is once in a lifetime event, can as well log it, might help with debugging.

Reply:
done